server/dbClients/articlesDB.js

```diff
@@ -1,18 +1,21 @@
 const mongoConnection = process.env.MONGODB_URI || 'mongodb://localhost:27017/legodi';
 const Article = require('../models/Article');
 const mongoose = require('mongoose')
-mongoose.Promise=global.Promise;
+mongoose.Promise = global.Promise;
 
- const addNewArticle = (query, callback) => {
+const addNewArticle = (query, callback) => {
     mongoose.connect(mongoConnection);
     Article.create(query).then(callback)
 };
 
-const listArticles = (sucessCallBack) => {
+const findArticles = (query, sucessCallBack) => {
```

Comment:
As @rarmatei pointed out, for `mongoose.connect` in both clients - Please take them out in a separate file called *connection.js* then require that file in top of these dbClients. That should be enough to start a connection, no need to connect every time.
Look at this [PR](https://github.com/CodeYourFuture/eribognal/pull/24/files#diff-c3cca03506a73628ce26de5dd6655bc2) to see how it can be done